torchaudio/transforms/_transforms.py

```diff
@@ -1248,6 +1248,32 @@ def __init__(self, time_mask_param: int, iid_masks: bool = False, p: float = 1.0
         super(TimeMasking, self).__init__(time_mask_param, 2, iid_masks, p=p)
 
 
+class Loudness(torch.nn.Module):
+    r"""Measure audio loudness according to the ITU-R BS.1770-4 recommendation.
+
+    .. devices:: CPU CUDA
+
+    .. properties:: TorchScript
+
+    Reference:
+        - https://www.itu.int/rec/R-REC-BS.1770-4-201510-I/en
+    """
+
+    def __init__(self):
```

Comment:
let's move the `sample_rate` parameter into `__init__`, set `self.sample_rate`, and move the docstrings up too